deploy-board/deploy_board/webapp/helpers/baseimages_helper.py

```diff
@@ -72,6 +90,8 @@ def get_acceptance_by_name(request, name, cell_name):
     params = [('cellName', cell_name)]
     return rodimus_client.get("/base_images/acceptances/%s" % name, request.teletraan_user_id.token, params=params)
 
+def get_golden_tag_by_name_and_cell(request, name, cell):
```

Comment:
`get_current_golden_image` so it matches the Rodimus API

Reply:
ok